src/sentry/integrations/bitbucket/client.py

```diff
@@ -58,7 +58,7 @@ def request(self, method, path, data=None, params=None, **kwargs):
             "sub": self.subject,
         }
         encoded_jwt = jwt.encode(jwt_payload, self.shared_secret)
-        headers = {"Authorization": b"JWT %s" % encoded_jwt}
+        headers = {"Authorization": "JWT %s" % encoded_jwt}
```

Comment:
I'm not sure this change is needed - I'm still running into the issue I mentioned before, but if I put this back to bytes it works.

Reply:
this sounds like you forgot to re-install the dependencies?  On the new version the `encoded_jwt` is a string and hence this was changed.  I don't think there's much to gain by supporting both versions at the same time so would be prefer to leave it like this.